### DISCUSSION TITLE
I stopped earning pp

### DISCUSSION THREAD

**Discussion by Shirou27** [ppy/osu#22352] [Q&A] [answered]
After the update i can't earn pp anymore it just has a warning sign and even there's a pp count it still doesn't add to my profile pp
![Screenshot_2023-01-22-21-28-45-712_sh ppy osulazer](https://user-images.githubusercontent.com/105548541/213919118-93c4baed-2ad9-44e4-8b32-85fa8c70d653.jpg)

My lazer profile link: https://lazer.ppy.sh/users/23892930

**Comment by bdach** [comment]
Warning signs mean that the scores are processing. This will likely resolve itself in due time.

**Comment by bdach** [comment]
We've had two reports of this now (this and https://github.com/ppy/osu/discussions/22398). @peppy any indications on the infra side as to whether something may be going wrong about this? I'm looking at https://status.ppy.sh/ but I'm not sure there are any visible metrics about processing faults to interpret there.

**Reply by peppy** [reply]
I have had some high error rate triggers, will be investigating soon.

We don't have enough exposed statistics on status site for lazer queue/pp systems yet.

**Comment by peppy** [comment]
@Shirou27 what platform / game version are you running?

**Reply by Shirou27** [reply]
I'm playing on Android platform, i forgot what game version I'm using that day. Sorry for late reply

**Comment by peppy** [comment, marked as answer]
This looks to be related to users with a lot of scores, toppling the processor due to it considering non-pass scores as well.

Have rolled out a hotfix, but also https://github.com/ppy/osu-queue-score-statistics/pull/131 will improve this situation going forward.

@Shirou27 setting a new score should update your PP.
